Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -1350,7 +1350,7 @@ def setup_(self, step, stepName, stepDict, k, properties):
     reco = {
         # skip the @pixelTrackingOnlyValidation which cannot run together with the full reconstruction
         '-s': 'RAW2DIGI:RawToDigi+RawToDigi_pixelOnly,L1Reco,RECO:reconstruction+reconstruction_pixelTrackingOnly,RECOSIM,PAT,VALIDATION:@standardValidation+@miniAODValidation,DQM:@standardDQM+@ExtraHLT+@miniAODDQM+@pixelTrackingOnlyDQM',
-        '--procModifiers': 'alpaka,pixelNtupletFit',
+        '--procModifiers': 'alpaka',
```

Comment:
will this still run the pixel ntuplet reconstruction ?

Reply:
Yep. Since the transition to Alpaka all the chain is activateed just using the `alpaka` modifier. The `pixelNtupletFit` one was there because we had the two different chains CPU (from clusters) and GPU (from RAW) and needed a way to distinguish them. When we clean up the CUDA part, `pixelNtupletFit`  can go.